### DISCUSSION TITLE
Can helix resize window's size?

### DISCUSSION THREAD

**Discussion by Squirre17** [helix-editor/helix#5618] [Q&A] [answered]
like something ctrl-h ctrl-l to resize in nvim/vim

**Comment by kirawi** [comment, marked as answer]
https://github.com/helix-editor/helix/pull/2704

**Reply by zeroishero** [reply]
the pr has been closed. Will it not be added to helix?

**Reply by sicher** [reply]
It's not closed but marked "inactive" and "waiting for author".
